app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -95,11 +97,25 @@ private void addBuildingMarkers() {
         for (Building building : loyBuildings) {
             createBuildingMarkersAndPolygonOverlay(building);
         }
+        map.setOnPolygonClickListener(new GoogleMap.OnPolygonClickListener() {
+            @Override
+            public void onPolygonClick(Polygon polygon) {
+                ((MainActivity) getActivity()).createToast("Building Clicked");
```

Comment:
I think these may be useful for debugging purposes... but are we leaving them in Master?